src/Compilers/Core/MSBuildTask/Directory.Build.props

```diff
@@ -0,0 +1,71 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the MIT license.  See License.txt in the project root for license information. -->
+<Project>
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+
+  <PropertyGroup>
+    <DefaultLanguage>en-US</DefaultLanguage>
```

Comment:
What is the effect of this property?

Reply:
I honestly do not know but it was present in the original task project hence copied it here for the time being